index.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>BusMall Orders</title>
+  <meta name="description" content="BusMall catalogue">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.6.0/Chart.min.js"></script>
+  <link href="https://fonts.googleapis.com/css?family=Monoton|Russo+One" rel="stylesheet">
+</head>
+
+<header>
+    <img src="img/bus-mall-logo.jpg" id="BusMall logo" title="BusMall logo">
+    <h1>BusMall</h1>
+</header>
+
+<body>
+    <main>
+        <h2 id="contactTitle">Please place your order.</h2>
+        <form id="contactInfo" class="forms">
+            <fieldset>
+                <legend>Contact information</legend>
+                <label for="userName"> name: </label>
+                <input name="userName" type="text" size="20" /><br>
+          
+                <label for="street">Street: </label>
+                <input name="street" type="text" size="20"><br>
+          
+                <label for="city">City: </label>
+                <input name="city" type="text" size="20"><br>
+          
+                <label for="state">State:</label>
+                <input name="state" type="text" size="20"><br>
+
+                <label for="zipcode">zipcode:</label>
+                <input name="zipcode" type="number"><br>
+
+                <label for="phoneNumber">Phone Number:</label>
+                <input name="phoneNumber" type="text"><br>
+          
+                <button type="submit">Submit</button>
+              </fieldset>
+        </form>
+
+        <h2>Shopping List</h2>
+        <form id="shoppingList" class="forms">
+            <fieldset id="innerShoppingList">
+                <select id="productList">
+                    <option value="select">- select -</option>
+                    <option value="bag">Bag</option>
+                    <option value="banana">Banana</option>
+                    <option value="bathroom">Bathroom</option>
+                    <figure>
+                    </figure>
+                </select>
```

Comment:
You also need a `input` so that the user can enter how many of the item they want to order. Also, a submit button to add it to the shopping cart.